src/sass/_sections.scss

```diff
@@ -5,7 +5,10 @@ section,
   .inner {
     margin: 0 24px;
     @media(min-width: 768px) {
```

Comment:
Can we re-use the breakpoint variables in `src/sass/base/_variables.scss` here? In this case we'd replace 768px with $tablet-screen. 
This will make it easier to adjust breakpoints site-wide if we need to.